commons/commands.py

```diff
@@ -153,6 +153,17 @@
 IPMI_SDR_TYPE_CMD = "ipmitool sdr type"
 IPMI_EVENT_CMD = "ipmitool event"
 
+#Provisioner AWS Commands
+AWS_INSTALL = "pip3 install awscli"
+AWS_PLUGIN = "pip3 install awscli-plugin-endpoint"
+AWS_VERSION = "aws --version"
+
+#IOs Opearations
+COPY_BUCKETFILE = "aws s3 cp file s3://test-bucket --endpoint-url http://s3.seagate.com:{}"
+GET_OBJECT = "aws s3api get-object --bucket test-bucket --key file --endpoint http://s3.seagate.com:{} outfile"
```

Comment:
Please revisit the existing commands from line 312 -325, they are same just you need to build your command using concatenation of endpoint flag and its value.do not write duplicate commands.